pages/linux/kscreen-doctor.md

```diff
@@ -0,0 +1,16 @@
+# kscreen-doctor
+
+> KScreen Doctor allows changing the screen setup from the command-line.
+> More information: <https://invent.kde.org/plasma/libkscreen>.
+
+- Show display output information:
+
+`kscreen-doctor --outputs`
+
+- Set the rotation of a display output with an ID of 1 to the right:
+
+`kscreen-doctor {{output.1.rotation.right}}`
+
+- Set the scale of a display output with an ID of HTML-2 to 2 (200%):
```

Comment:
typo?
```suggestion
- Set the scale of a display output with an ID of `HDMI-2` to 2 (200%):
```

Reply:
No, one of my monitors is just plugged into my HTML port on my laptop. 😆 
Not sure how I managed that one.